syncd/Syncd.cpp

```diff
@@ -2436,18 +2478,35 @@ void Syncd::processFlexCounterGroupEvent( // TODO must be moved to go via ASIC c
 
     consumer.pop(kco);
 
-    auto& groupName = kfvKey(kco);
+    auto& key = kfvKey(kco);
     auto& op = kfvOp(kco);
     auto& values = kfvFieldsValues(kco);
 
-    WatchdogScope ws(m_timerWatchdog, op + ":" + groupName, &kco);
+    processFlexCounterGroupEvent(key, op, values, false);
+}
+
+void Syncd::processFlexCounterGroupEvent( // TODO must be moved to go via ASIC channel queue
+        _In_ const std::string &groupName,
+        _In_ const std::string &op,
+        _In_ const std::vector<swss::FieldValueTuple> &values,
+        _In_ bool operateDb)
+{
+    WatchdogScope ws(m_timerWatchdog, op + ":" + groupName);
```

Comment:
Why we suddenly need wathdog here, wathdog was used only for SAI apps here we dont have SAI callz

Reply:
Removed.